ios/Video/RCTVideo.m

```diff
@@ -828,15 +828,13 @@ - (void)setIgnoreSilentSwitch:(NSString *)ignoreSilentSwitch
 - (void)setPaused:(BOOL)paused
 {
   if (paused) {
-    [_player pause];
     [_player setRate:0.0];
   } else {
     if([_ignoreSilentSwitch isEqualToString:@"ignore"]) {
       [[AVAudioSession sharedInstance] setCategory:AVAudioSessionCategoryPlayback error:nil];
     } else if([_ignoreSilentSwitch isEqualToString:@"obey"]) {
       [[AVAudioSession sharedInstance] setCategory:AVAudioSessionCategoryAmbient error:nil];
     }
-    [_player play];
```

Comment:
This pause-then-play-again thing is... interesting. Looks like it was introduced in some super-poorly-documented commit back in 2015:
https://github.com/react-native-community/react-native-video/commit/f2b0dbd2db81f2e03f2cee457eb4e465ec111b76

...so yeah, sounds good to me to remove this hacky bit!